pkg/cmd/admin/router/router.go

```diff
@@ -239,32 +239,6 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out io.Writer)
 	return cmd
 }
 
-// Read the specified file and return it as a bytes array.
-func loadData(file string) ([]byte, error) {
-	if len(file) == 0 {
-		return []byte{}, nil
-	}
-
-	bytes, err := ioutil.ReadFile(file)
-	if err != nil {
-		return []byte{}, err
-	}
-
-	return bytes, nil
-}
-
-// Read the specified certificate file and return it as a string.
-func loadCert(file string) (string, error) {
-	bytes, err := loadData(file)
-
-	return string(bytes), err
-}
-
-// Read the specified key file and return it as a bytes array.
-func loadKey(file string) ([]byte, error) {
-	return loadData(file)
-}
-
```

Comment:
Been meaning to do this for a while. Thx for pulling this code out. 
Minor nit, would have liked it in a separate PR though as it doesn't have much bearing to this one.